build.gradle

```diff
@@ -43,11 +42,11 @@ dependencies {
 	testAnnotationProcessor 'org.projectlombok:lombok'
 
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
-	testImplementation 'com.tngtech.archunit:archunit-junit5:1.3.0'
-	testImplementation 'net.datafaker:datafaker:2.4.2'
-	testImplementation 'org.junit.jupiter:junit-jupiter:5.12.2'
-	testImplementation 'org.testcontainers:junit-jupiter:1.20.6'
-	testImplementation 'org.testcontainers:postgresql:1.20.6'
+	testImplementation 'com.tngtech.archunit:archunit-junit5:1.4.1'
```

Comment:
Its not required but its nice to move versions to gradle.properties, example of what You put inside
<img width="269" height="164" alt="image" src="https://github.com/user-attachments/assets/e589b014-ee57-46cb-9d14-ae23980c6215" />

And then You can reference them here like that:

`"org.testcontainers:postgresql:${postgresqlVersion}"`